pkg/kubectl/cmd/taint.go

```diff
@@ -351,23 +346,18 @@ func (o TaintOptions) RunTaint() error {
 }
 
 // validateNoTaintOverwrites validates that when overwrite is false, to-be-updated taints don't exist in the node taint list (yet)
-func validateNoTaintOverwrites(accessor metav1.Object, taints []v1.Taint) error {
-	annotations := accessor.GetAnnotations()
-	if annotations == nil {
-		return nil
+func validateNoTaintOverwrites(obj runtime.Object, taints []v1.Taint) error {
+	node, ok := obj.(*v1.Node)
+	if !ok {
+		return fmt.Errorf("unexpected type %T", obj)
```

Comment:
can you add at the end of the message "expected Node" (though it will be obvious if someone traces down the source code line)